Examples/Tests/collision/inputs_test_1d_collision_z_Bremsstrahlung

```diff
@@ -0,0 +1,106 @@
+#################################
+########## CONSTANTS ############
+#################################
+
+my_constants.n_e = 5.47e30     # electron density, m^-3
+my_constants.Np_e = 1024       # electrons per cell
+my_constants.T1 = 1.e6         # 1 MeV, electron energy
+my_constants.m_e_eV = m_e*clight**2/q_e # electron rest mass in eV
+my_constants.gamma = T1/m_e_eV + 1. # electron gamma factor
+my_constants.gamma_beta = sqrt(gamma**2 - 1.)
+
+my_constants.n_i = 5.47e31     # ion density, m^-3, for rho = 1000 g/cm^3
+my_constants.Np_i = 1024       # ions per cell
+my_constants.T_i = 2.0         # ion temperature, eV
+
+my_constants.m_B11 = 20065.0*m_e  # 11.01*amu/me - 5, Boron
+my_constants.q_B11 = 5.*q_e
+
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+max_step = 100
+amr.n_cell = 100
+amr.max_level = 0
+amr.blocking_factor = 4
+geometry.dims = 1
+geometry.prob_lo = 0.
+geometry.prob_hi = 1.e-6
+
+#################################
+###### Boundary Condition #######
+#################################
+boundary.field_lo = periodic
+boundary.field_hi = periodic
+
+#################################
+############ NUMERICS ###########
+#################################
+warpx.serialize_initial_conditions = 1
+warpx.verbose = 1
+warpx.const_dt = 1.e-2*1.e-15
+warpx.use_filter = 0
+
+# Do not evolve the E and B fields
+algo.maxwell_solver = none
+
+# Order of particle shape factors
+algo.particle_shape = 1
+
+#################################
+############ PLASMA #############
+#################################
+particles.species_names = electrons ions photons
+particles.photon_species = photons
+
+electrons.species_type = electron
+electrons.do_not_deposit = 1
+
+electrons.injection_style = "NUniformPerCell"
+electrons.num_particles_per_cell_each_dim = Np_e
+electrons.profile = "constant"
+electrons.density = n_e
+electrons.momentum_distribution_type = "constant"
+electrons.uz = gamma_beta
+
+ions.charge = q_B11
+ions.mass = m_B11
+ions.do_not_deposit = 1
+
+ions.injection_style = "NUniformPerCell"
+ions.num_particles_per_cell_each_dim = Np_i
+ions.profile = "constant"
+ions.density = n_i
+ions.momentum_distribution_type = "gaussian"
+ions.ux_th = sqrt(T_i*q_e/m_B11)/clight
+ions.uy_th = sqrt(T_i*q_e/m_B11)/clight
+ions.uz_th = sqrt(T_i*q_e/m_B11)/clight
+
+photons.species_type = photon
+photons.injection_style = none
+
+#################################
+############ COLLISION ##########
+#################################
+collisions.collision_names = bremsstrahlung
+bremsstrahlung.type = bremsstrahlung
+bremsstrahlung.species = electrons ions
+bremsstrahlung.Z = 5
```

Comment:
Is this to overwrite the ionization state or, since this is a particle-particle mode, can we also rely on the charge / ionization state of the ion we collide with?

Reply:
The `Z` is only used to indicate the atomic number, which is used to select which cross section data to use from the tables.